src/client/index.jsx

```diff
@@ -34,11 +34,11 @@ const Main = () => (
               <Route exact path="/" component={Health} />
               <Route exact path="/engine" component={Engine} />
               <Route exact path="/south" component={South} />
-              <Route exact path="/south/:dataSourceId" component={ConfigureProtocol} />
-              <Route exact path="/south/:dataSourceId/live" component={SouthStatus} />
-              <Route exact path="/south/:dataSourceId/points" component={ConfigurePoints} />
+              <Route exact path="/south/:id" component={ConfigureProtocol} />
```

Comment:
same question. should the id or the name be visible in the path

Reply:
The url is used to identify the connector, hence the use of the id instead of the name. Moreover, with an Id in the url, we have the same pattern than OIAnalytics to identify entities (dashboards, users,...)